libvisual/CMakeLists.txt

```diff
@@ -146,8 +146,8 @@ ENDIF()
 
 # Build profiles
 # Standard
-SET(CMAKE_C_FLAGS "${OPT_C_FLAGS} ${PROFILE_C_FLAGS} -std=c99 -Wall -Wextra -Wunused ${WERROR_C_FLAGS} -fstrict-aliasing")
-SET(CMAKE_CXX_FLAGS "${OPT_C_FLAGS} ${PROFILE_C_FLAGS} ${CXX11_FLAGS} -Wall -Wextra -Wunused ${WERROR_C_FLAGS} -fstrict-aliasing")
+SET(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} ${OPT_C_FLAGS} ${PROFILE_C_FLAGS} -std=c99 -Wall -Wextra -Wunused ${WERROR_C_FLAGS} -fstrict-aliasing")
+SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${OPT_C_FLAGS} ${PROFILE_C_FLAGS} ${CXX11_FLAGS} -Wall -Wextra -Wunused ${WERROR_C_FLAGS} -fstrict-aliasing")
```

Comment:
Is this about supporting flags provided by the user? If so, I'd expect them to go last so they can override `${OPT_C_FLAGS}` and friends. Why this way around? What was this addition needed for?

Reply:
This addition was originally there since I needed it for adding another lib, but it was eventually nullified by a different CMake modification. I just added it to the beginning due to a habit with other CMakeLists, I will move it over